sjsonnet/src-jvm-native/sjsonnet/Config.scala

```diff
@@ -123,4 +123,9 @@ case class Config(
     doc = "Evaluate the given string as Jsonnet rather than treating it as a file name"
   )
   exec: Flag = Flag(),
+  @arg(
+    name = "no-duplicate-keys-in-comprehension",
+    doc = "Raise an error if an object comprehension contains duplicate keys"
+  )
+  noDuplicateKeysInComprehension: Flag = Flag(),
```

Comment:
Can we make the correct behavior the default? We did the same when we added static checking.

Reply:
This is mostly to de-risk the version bump on the Databricks side of things. I'd rather make this opt-in and then go and turn it on piece-meal in a controlled fashion, rather than making it opt-out and then worrying about whether there's some place where we forgot to opt out that may blow up during the upgrade